components/fonts/font_context.rs

```diff
@@ -78,8 +78,7 @@ pub struct FontContext {
     /// A caching map between the specification of a font in CSS style and
     /// resolved [`FontGroup`] which contains information about all fonts that
     /// can be selected with that style.
-    resolved_font_groups:
-        RwLock<HashMap<FontGroupCacheKey, FontGroupRef, BuildHasherDefault<FnvHasher>>>,
+    resolved_font_groups: RwLock<FxHashMap<FontGroupCacheKey, FontGroupRef>>,
```

Comment:
I am also not sure about this one, it uses https://doc.servo.org/style/properties/generated/style_structs/struct.Font.html#structfield.hash, but I am not sure if it's browser generated or can it be influenced by user input.